app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -69,37 +62,31 @@ If your application is successful, the training provider may be able to help you
 
 Most schools in England need you to have [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) to teach. 
 
-Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/train-to-be-a-teacher/what-is-a-pgce), and some will award both. 
+Without QTS, you will not be eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.
```

Comment:
I know this is in the original content but I wonder if it is slightly misleading on two counts:

* scholarships and bursaries are only available for certain subjects
* scholarships and bursaries are for courses leading to QTS - the way it is written implies you need QTS first?

Is there is a better way of wording this?

And once we've done that, it _may_ make more sense to return this section to the original order?